modules/administration-guide/partials/proc_setting-up-the-github-oauth-app.adoc

```diff
@@ -0,0 +1,52 @@
+:_content-type: PROCEDURE
+:description: Setting up the GitHub OAuth App
+:keywords: github, github-oauth-app
+:navtitle: Setting up the GitHub OAuth App
+// :page-aliases:
+
+[id="setting-up-the-github-oauth-app_{context}"]
+= Setting up the GitHub OAuth App
+
+Set up a GitHub OAuth App using OAuth 2.0.
+
+.Prerequisites
+
+* You are logged in to GitHub.
+* link:https://www.gnu.org/software/coreutils/base64[`base64`] is installed in the operating system you are using.
+
+.Procedure
+
+. Go to link:https://github.com/settings/applications/new[].
+//Long version: Click your GitHub avatar menu:Settings[Developer settings > OAuth Apps > Register a new application]. max-cx
+
+. Enter *{prod-short}* as the *Application name*.
+
+. Enter `pass:c,a,q[{prod-url}]/` as the *Homepage URL*.
+
+. Enter `pass:c,a,q[{prod-url}]/api/oauth/callback` as the *Authorization callback URL*.
+
+. Click *Register application*.
```

Comment:
Consider concatenating in one single step:

. Go to link:https://github.com/settings/applications/new[the Register a new OAuth application form], fill the form, and click *Register application*. Use following values:

(here a key / value table)

Reply:
Thanks for the suggestion.
Unfortunately, this approach doesn't seem to work for single-line steps.
Currently, I have only three single-line steps.
If grouping these steps into one, I get one more line even though the step count is reduced:
![image](https://user-images.githubusercontent.com/78564304/162999360-62893377-7a27-46d2-8fe8-b098b7787068.png)